psi4/driver/schema_wrapper.py

```diff
@@ -185,10 +214,60 @@ def _read_output(outfile):
 
         return output
     except OSError:
-        return "Could not read output file"
+        return "Could not read output file."
+
+
+def _quiet_remove(filename):
+    """
+    Destroy the created at file at exit, pass if error.
+    """
+    try:
+        os.unlink(filename)
+    except OSError:
+        pass
+
+
+def run_qcschema(input_data, clean=True):
+
+    outfile = os.path.join(core.IOManager.shared_object().get_default_path(), str(uuid.uuid4()) + ".qcschema_tmpout")
+    core.set_output_file(outfile, False)
+    print_header()
+    start_time = datetime.datetime.now()
+
+    try:
+        input_model = qcng.util.model_wrapper(input_data, qcel.models.ResultInput)
+
+        # qcschema should be copied
+        ret_data = run_json_qcschema(input_model.dict(), clean, False)
+        ret_data["provenance"] = {"creator": "Psi4", "version": __version__, "routine": "psi4.schema_runner.run_qcschema"}
+
+        exit_printing(start_time=start_time, success=True)
+
+        ret = qcel.models.Result(**ret_data, stdout=_read_output(outfile))
+
+    except Exception as exc:
+
+        input_data = input_data.copy()
+        input_data["stdout"] = _read_output(outfile)
+        ret = qcel.models.FailedOperation(input_data=input_data,
+                                          success=False,
+                                          error={
+                                              'error_type': type(exc).__name__,
+                                              'error_message': ''.join(traceback.format_exception(*sys.exc_info())),
+                                          })
+
+    atexit.register(_quiet_remove, outfile)
+
+    return ret
+
 
 def run_json(json_data, clean=True):
 
+    warnings.warn(
+        "Using `psi4.schema_wrapper.run_qcschema` instead of `psi4.json_wrapper.run_json` is deprecated, and in 1.5 it will stop working\n",
```

Comment:
Should this be `run_qcschema` or `run_json` that's being deprecated? Or does the `stacklevel=2` ensure the warning is only printed when `run_json` is called from `run_qcschema` via here:

https://github.com/psi4/psi4/blob/667c7e7fa67105237fb8a84bd2810bcc097d7f3d/psi4/run_psi4.py#L262

as opposed to a `stacklevel=1` for the call here:

https://github.com/psi4/psi4/blob/667c7e7fa67105237fb8a84bd2810bcc097d7f3d/psi4/run_psi4.py#L288

?

Reply:
Whoops, fixed.